lua/obsidian/commands/checkbox.lua

```diff
@@ -0,0 +1,14 @@
+local toggle_checkbox = require("obsidian.util").toggle_checkbox
+
+---@param client obsidian.Client
+return function(client)
+  local checkboxes = vim.tbl_keys(client.opts.ui.checkboxes)
+  for k, v in pairs(client.opts.ui.checkboxes) do
+    local order = v.order
+    if order and type(order) == "number" then
+      -- sort the checkboxes based on order
+      checkboxes[order] = k
+    end
+  end
```

Comment:
I think sorting here based on order would be more robust. Something like:

```suggestion
  table.sort(checkboxes, function(a, b)
    return client.opts.ui.checkboxes[a].order < client.opts.ui.checkboxes[b].order
  end)
```

Reply:
Thanks for the advice! I'll commit the changes